content/posts/2023/reana-client-prune-command.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Free up disk space in REANA workspaces with the new prune command"
+date: 2023-04-24T07:00:00+01:00
+---
+
+The new prune command in the reana-client software allows users to easily
+free up disk space in their REANA workspaces by deleting temporary files
+that are neither inputs nor outputs. This can help manage cluttered
+workspaces and prevent running out of disk space. 
+
+<!--more-->
+
+
+## Enable Kerberos for a single workflow step
+
+The management of disk space in REANA can be a challenge, especially when the workspace is
+cluttered by numerous temporary files that are no longer needed.
+If you have been looking for a way to easily free up disk space in your workspaces,
+look no further than the new `prune` command in `reana-client`!
+
+With the `prune` command, you can easily delete all the files from your workspace that are
+neither input nor output files. This can free up a significant amount of disk space and make
+your workspace much more manageable. Plus, if you need to free up even more space, you can use the
+`--include-inputs` and `--include-outputs` flags to delete those kinds of files too.
```

Comment:
Okay, I see your point and I like the suggestion. I tried to rewrite the first paragraphs, so that they are just an introduction and include this common scenario. What do you think?

Reply:
I would simplify the text a bit more, and especially add a warning that people should properly populate their `outputs` clause of `reana.yaml` specification file, otherwise the prune command may remove "too much"! I saw many people don't bother populating "outputs" because it does not really have had any impact besides ease of [downloading many files](https://forum.reana.io/t/reana-client-download-all-files/70). So we should really underline this here.